src/tcgwars/logic/impl/gen3/DeltaSpecies.groovy

```diff
@@ -2669,15 +2716,15 @@ public enum DeltaSpecies implements LogicCardInfo {
         text "You can play only one Supporter card each turn. When you play this card, put it next to your Active Pokémon. When your turn ends, discard this card." +
           "Discard a card from your hand. If you can't discard a card from your hand, you can't play this card. Search your discard pile for 3 basic Energy cards and any combination of 3 Basic Pokémon or Evolution cards, show them to your opponent, and put them on top of your deck. Shuffle your deck afterward."
         onPlay {
-          def toDiscard = my.hand.getExcludedList(thisCard).select(count:1, "Discard a card to discard.")
+          def toDiscard = my.hand.getExcludedList(thisCard).select(count:1, "Discard a card from your hand in order to play ${thisCard}.")
           toDiscard.discard()
 
           if (my.discard.filterByType(BASIC_ENERGY)) {
-            my.discard.filterByType(BASIC_ENERGY).select(max: 3, "Select 3 Basic Energies to shuffle into your deck").moveTo(my.deck)
+            my.discard.filterByType(BASIC_ENERGY).select(count: 3, "Select 3 Basic Energies to shuffle into your deck").moveTo(my.deck)
```

Comment:
Does this work if you only have 2 Basic Energies in discard?

Reply:
Yeah, it's used by some other cards already. Just tested on Energy Recycler (Guardians Rising) and it took less than 5 basic energies fine :)